kolibri/core/content/utils/channel_import.py

```diff
@@ -951,7 +952,12 @@ def set_learning_activities_from_kind(self, ContentNodeTable):
         for kind, la in kind_activity_map.items():
             self.destination.execute(
                 ContentNodeTable.update()
-                .where(ContentNodeTable.c.kind == kind)
+                .where(
+                    and_(
```

Comment:
This looks like precisely the correct change, and there is no reason this shouldn't have been here previously, except for my oversight in not including it.